spiffworkflow/providers.tf

```
@@ -0,0 +1,13 @@
+terraform {
+  required_version = "~> 1.0"
+  required_providers {
+    cloudfoundry = {
+      source  = "cloudfoundry/cloudfoundry"
+      version = ">=1.1.0"
```

[suggestion] there have been important bug fixes in recent versions, so I'd start this at the most recent available today:

```suggestion
      version = ">=1.3.0"
```